packages/vscode-extension/package.nls.json

```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"It seems that the tunnel in your account has exceeded its limit. Would you like to delete all tunnels created through Teams Toolkit?",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelListMessage":"The following is the dev tunnels created by Teams Toolkit in your account:",
```

Comment:
You have reached maximum numbers of tunnel created with your Microsoft 365 account. Clean up tunnels to continue debugging your Teams application. Refer to Output channel to review tunnel details.